plugins/workspace-plugin/src/git.ts

```diff
@@ -45,16 +45,16 @@ export async function getRemoteURL(remote: string, projectPath: string): Promise
 export async function sparseCheckout(
   projectPath: string,
   repositoryUri: string,
-  sparseCheckoutDirectory: string,
+  sparseCheckoutDirectories: string[],
   commitReference: string
 ): Promise<void> {
   await initRepository(projectPath);
   // Enable sparse checkout feature
   await setConfig(projectPath, 'core.sparsecheckout', 'true');
   // Write sparse checkout directory
   const gitInfoFolderPath = path.join(projectPath, '.git/info/');
-  fs.ensureDirSync(gitInfoFolderPath);
-  fs.writeFileSync(path.join(gitInfoFolderPath, 'sparse-checkout'), sparseCheckoutDirectory);
+  await fs.pathExists(gitInfoFolderPath);
```

Comment:
I suppose we need to use `ensureDir` or `mkdirs` here, as `pathExists` checks only the existence
https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/pathExists.md